spec/classes/plugin/remote_execution_cockpit_spec.rb

```diff
@@ -71,7 +71,7 @@ class {'foreman':
             .without_content
             .with_ssl_content(%r{^<Location /webcon>$})
             .with_ssl_content(%r{^  RewriteRule /webcon/\(\.\*\)           ws://127\.0\.0\.1:19090/webcon/\$1 \[P\]$})
```

Comment:
This is still needed? And for websockets we don't suffer from the same problem? I ask because I get the impression this line is redundant

Reply:
My understanding is (without having explicitly tested that) that w/o this we do not get a protocol upgrade on EL8.

On EL9 (and Debian), we could use [`ProxyPass … upgraade=websocket`](https://httpd.apache.org/docs/2.4/mod/mod_proxy.html#wsupgrade), but not on EL8